core/app/models/spree/product.rb

```diff
@@ -127,7 +127,7 @@ class Product < Spree::Base
 
     attr_accessor :option_values_hash
 
-    accepts_nested_attributes_for :product_properties, allow_destroy: true, reject_if: ->(pp) { pp[:property_name].blank? }
```

Comment:
@aleksandarpetrushev let's leave is at is, and just add HTML5 required tag on the dashboard please

Reply:
@damianlegawiec https://github.com/spree/spree_backend/pull/79